src/seedsigner/views/seed_views.py

```diff
@@ -1398,22 +1400,22 @@ def run(self):
                 sig_type = SettingsConstants.MULTISIG
                 if self.controller.multisig_wallet_descriptor:
                     # Can jump straight to the brute-force verification View
-                    destination = Destination(SeedAddressVerificationView)
+                    destination = Destination(SeedAddressVerificationView, skip_current_view=True)
                 else:
                     self.controller.resume_main_flow = Controller.FLOW__VERIFY_MULTISIG_ADDR
-                    destination = Destination(LoadMultisigWalletDescriptorView)
+                    destination = Destination(LoadMultisigWalletDescriptorView, skip_current_view=True)
 
             else:
                 sig_type = SettingsConstants.SINGLE_SIG
-                destination = Destination(SeedSingleSigAddressVerificationSelectSeedView)
+                destination = Destination(SeedSingleSigAddressVerificationSelectSeedView, skip_current_view=True)
 
         elif self.controller.unverified_address["script_type"] == SettingsConstants.TAPROOT:
             # TODO: add Taproot support
-            return Destination(NotYetImplementedView)
+            return Destination(NotYetImplementedView, skip_current_view=True)
 
         elif self.controller.unverified_address["script_type"] == SettingsConstants.LEGACY_P2PKH:
             # TODO: detect single sig vs multisig or have to prompt?
-            return Destination(NotYetImplementedView)
+            return Destination(NotYetImplementedView, skip_current_view=True)
```

Comment:
This View should not be added to the backstack (see [here](https://github.com/SeedSigner/seedsigner/pull/286#discussion_r1056014645) for discussion).

Reply:
TY @EverydayBitcoiner, The top 3 changes to skip_current_view=True from this View will be borrowed and included in a smaller pr.  I'll edit this later.  In all 3 cases, these additions allow the user to exit to the main menu via the back button, instead of landing in the scan view for the first two, or landing in the same screen forever in the last one.

Note for adding ", skip_current_view=True" to Destination(NotYetImplementedView) is currently NEVER required because regardless of how the user exits that screen, via "ok" or "back" the next step is directly to the MainMenuView.